### PR TITLE
Add support for allowed_ports field to terraform

### PR DESCRIPTION
**Pull request by tejal29**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
Add support for allowed_ports field to the workstation config.
```

### CODE REVIEW

File: mmv1/templates/terraform/examples/workstation_config_basic.tf.erb [hunk 1]
```diff
@@ -69,4 +69,18 @@ resource "google_workstations_workstation_config" "<%= ctx[:primary_resource_id]
       }
     }
   }
+  allowed_ports = [
```
Comment (hao-nan-li) [inline mmv1/templates/terraform/examples/workstation_config_basic.tf.erb:72]: Might have missing comma here according to the error message.

**Comment by hao-nan-li** [comment]
Any update on fixing the tests?

**Comment by tejal29** [comment]
Thanks. Will work on this next week. Sorry for the delay.

**Comment by c2thorn** [comment]
Hi @tejal29 

Tomorrow, the Magic Modules repository is scheduled to undergo a language migration from Ruby to Go. You can view more details about this in our announcement here: https://github.com/hashicorp/terraform-provider-google/issues/19583 (or [go/mm-migration-announcement](go/mm-migration-announcement) if you are a Googler)

This open pull request may become incompatible due to most YAML and .erb files converting to Go-compatible files.

Our team (Magic Modules repository maintainers) has tooling to automatically convert changes to the new language, and we can prepare a new commit for this pull request that is compatible with the migration.

In order to push the new changes to your pull request, we would need to <ins>force push the commit to your fork's branch</ins>. Our tooling saves a backup branch before converting, so we could rollback or open a new pull request if needed. We would also work with you and the PR reviewer in the event additional changes are needed.

You also have the option to update the pull request yourself after the migration. You can view a [preview branch](https://github.com/GoogleCloudPlatform/magic-modules/tree/FEATURE-BRANCH-rewrite-switchover-preview) and [updated documentation](https://googlecloudplatform.github.io/magic-modules/reference/ruby-go-changes/) related to the migration changes.

We will take no action until we have your explicit permission to push changes to your fork's branch used for this pull request. Let me or your reviewer know if you have any further questions!

**Pull request closed without merging**